validation/jvm/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ HmdaDsl, CommonDsl, Result }
+
+/*
+ Timestamp must be numeric and in ccyymmddhhmm format
+ */
+object S028 extends CommonDsl with HmdaDsl {
+  def apply(ts: TransmittalSheet): Result = {
+    import scala.language.postfixOps
+    val timestamp = ts.timestamp
+    (timestamp is numeric) and (timestamp.toString is validTimestampFormat)
```

Comment:
In that case, do we want the parser to return a string for the timestamp? Then S028 could pass or fail.